static/_redirects

```diff
@@ -183,37 +182,39 @@
 /guides/launch/create-queue /guides/launch/setup-queue-advanced 301
 /guides/launch/docker /guides/launch/setup-launch-docker 301
 /guides/launch/getting-started /guides/launch/walkthrough 301
-/guides/launch/integrations/* /guides/launch 301
+/guides/launch/integrations/sagemaker /guides/launch 301
+/guides/launch/integrations /guides/launch 301
 /guides/launch/kubernetes /guides/launch/setup-launch-kubernetes 301
-/guides/launch/launch-faqs /support/index_launch 301
 /guides/launch/launch-jobs /guides/launch/add-job-to-queue 301
 /guides/launch/prerequisites /guides/launch/getting-started 301
 /guides/launch/run-agent /guides/launch/setup-agent-advanced 301
 /guides/launch/sagemaker /guides/launch/setup-launch-sagemaker 301
 /guides/launch/vertex /guides/launch/setup-vertex 301
 /guides/manage-runs /guides/runs/manage-runs 301
-/guides/model_registry/quickstart /guides/model_registry/walkthrough 301
-/guides/models/access_controls /guides/model_registry/access_controls 301
-/guides/models/automation /guides/model_registry/model-registry-automations 301
-/guides/models/notifications /guides/model_registry/notifications 301
-/guides/models/model-management-concepts /guides/model_registry/model-management-concepts 301
-/guides/model_registry/quickstart /guides/model_registry/walkthrough 301
-/guides/manage-runs /guides/runs/manage-runs 301
+/guides/models/access_controls /guides/core/registry/model_registry/access_controls 301
```

Comment:
Not working, goes to https://docs-1245.docodile.pages.dev/guides/core/registry/model_registry/access_controls